content/en/real_user_monitoring/mobile_and_tv_monitoring/integrated_libraries/ios.md

```diff
@@ -0,0 +1,52 @@
+---
+title: iOS and tvOS Libraries for RUM
+code_lang: ios
+type: multi-code-lang
+code_lang_weight: 10
+aliases:
+- /real_user_monitoring/ios/integrated_libraries/
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-ios
+  tag: "Source Code"
+  text: Source code for dd-sdk-ios
+- link: /real_user_monitoring/mobile_and_tv_monitoring/integrated_libraries
+  tag: Documentation
+  text: Integrated Libraries
+---
+
+This page lists integrated libraries you can use for iOS and tvOS applications.
+
+## Alamofire
+
+Starting from version `2.5.0`, the RUM iOS SDK can automatically track [Alamofire][1] requests.
+
+1. Configure RUM monitoring by following the [Setup][2] guide.
+2. Enable `URLSessionInstrumentation` for `Alamofire.SessionDelegate`:
+
+```swift
+import Alamofire
+import DatadogRUM
+
+URLSessionInstrumentation.enable(with: .init(delegateClass: SessionDelegate.self))
+```
+For additional information on sampling rate, distributed tracing, and adding custom attributes to tracked RUM resources, refer to "Automatically track network requests" in the [Advanced Configuration][4] documentation.
+
+## Apollo GraphQL
+
+Starting from version `2.5.0`, the RUM iOS SDK can automatically track [Apollo GraphQL][3] requests.
+
+1. Configure RUM monitoring by following the [Setup][2] guide.
+2. Enable `URLSessionInstrumentation` for `Apollo.URLSessionClient`:
+
+```swift
+import Apollo
+import DatadogRUM
+
+URLSessionInstrumentation.enable(with: .init(delegateClass: URLSessionClient.self))
```

Comment:
/improvement

`URLSessionClient` is very common name, let's namespace it/
```suggestion
URLSessionInstrumentation.enable(with: .init(delegateClass: Apollo.URLSessionClient.self))
```

Reply:
Done 👍. I also considered linking [`Apollo.URLSessionClient` doc](https://www.apollographql.com/docs/ios/networking/request-pipeline/#the-urlsessionclient-class) but decided to not do it as external links are likely to change making our docs out-of-sync.